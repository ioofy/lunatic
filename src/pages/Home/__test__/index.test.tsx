import { render, screen, waitFor } from "@testing-library/react";
import { Home } from "../index.page";

describe("Home Screen", () => {
  it("Render Home screen", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });

  it("check if Home screen renders", async () => {
    render(<Home />);

    await waitFor(() => screen.getByText("Home"));
    expect(screen.getByText("Home").innerHTML).toBe("Home");
  });
});
