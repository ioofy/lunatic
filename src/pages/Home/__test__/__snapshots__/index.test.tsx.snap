// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen Render Home screen 1`] = `
<div>
  <div>
    Home
  </div>
</div>
`;
